Sources/Kommunicate/Classes/KMConversationViewController.swift

```diff
@@ -178,7 +182,51 @@ open class KMConversationViewController: ALKConversationViewController {
         })
     }
 
-    @objc override open func pushNotification(notification: NSNotification) {
+    open override func addMessagesToList(_ messageList: [Any]) {
+        guard let messages = messageList as? [ALMessage] else { return }
+        messageArray = messages
+        count = 0
+        timer = Timer.scheduledTimer(timeInterval: 3, target: self, selector: #selector(loopOverMessageArray), userInfo: nil, repeats: true)
+        timer.fire()
+    }
+    
+    @objc func loopOverMessageArray() {
+        
+        if count >= messageArray.count {
+            timer.invalidate()
+            return
+        }
+        var filteredArray = [ALMessage]()
```

Comment:
I don't think this array contains more than 1 message at a time. If that's the case, then we can change the type.